SAR_retrieval_machinery.py

```diff
@@ -24,16 +24,16 @@ def wcm_jac_(A, V1, B, V2, R, alpha, C, theta=23):
     a roughness term and a moisture-controlled term. The soil moisture
     term can be interpreted in weird and wonderful manners once retrieved
     (eg inverting the dielectric constant)
-    This function returns the gradient for all parameters (A, B, 
+    This function returns the gradient for all parameters (A, B,
     V1, V2 and C)."""
     mu = np.cos(np.deg2rad(theta))
     tau = np.exp(-2 * B * V2 / mu)
     veg = A * V1 * (1 - tau)
```

Comment:
The `mu` modulation of `A*V1` is missing here, but present in the Jacobian (however, it should just use `mu`)

Reply:
done